packages/app-project/package.json

```diff
@@ -6,10 +6,10 @@
   "version": "0.0.1",
   "main": "index.js",
   "scripts": {
-    "build": "PANOPTES_ENV=${PANOPTES_ENV:-production} next build",
+    "build": "APP_ENV=${APP_ENV:-development} PANOPTES_ENV=${PANOPTES_ENV:-production} next build",
     "build-storybook": "build-storybook -s public",
-    "dev": "PANOPTES_ENV=${PANOPTES_ENV:-staging} node server/dev-server.js",
-    "dev:inspect": "PANOPTES_ENV=${PANOPTES_ENV:-staging} NODE_OPTIONS='--inspect' node server/dev-server.js",
+    "dev": "APP_ENV=${APP_ENV:-development} PANOPTES_ENV=${PANOPTES_ENV:-staging} node server/dev-server.js",
+    "dev:inspect": "APP_ENV=${APP_ENV:-development} PANOPTES_ENV=${PANOPTES_ENV:-staging} NODE_OPTIONS='--inspect' node server/dev-server.js",
```

Comment:
Looks good to me! What does `dev:inspect` do here?

Reply:
Oh ignore that, I see the description already in app-project's readme!

Reply:
It runs [the Node debugger](https://nodejs.org/en/docs/guides/debugging-getting-started/), which can be useful if you want to debug `getStaticProps` or `getServerSideProps`.

You can connect to it from the debugger in Chrome dev tools, although I don't remember how, off the top of my head. It was pretty easy to figure out, the last time I tried using it.